packages/bundle/README.md

```diff
@@ -0,0 +1,67 @@
+# @b-mo/bundle
+
+This packages loads and resolves a dependency bundle. It allows you to easily extend dependency bundles
```

Comment:
Should there be commas after packages and loads, or should packages be singular?